auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,127 @@
+
+ * Another way is to use 3rd-party Mesh Processing Software like [Meshlab](https://www.meshlab.net/). Such softwares take the mesh file as an input and provide the inertial parameters as an output which can then be copied and pasted into the URDF/SDF file. This is also the method that was suggested in official [Classic Gazebo docs](https://classic.gazebosim.org/tutorials?cat=build_robot&tut=inertia).
+
+Both of these ways create a dependency on external software and might be complicated for beginners. Integrating this directly in Gz Sim would provide a smoother and user-friendly workflow.
+
+## User Perspective
+
+Currently to specify the `<inertial>` element of a `<link>` in SDFormat, the user needs to add the `<mass>`, `<pose>`, and `<inertia>` tags. Here `<mass>` is the mass of the link and `<pose>` is the pose of the Centre of Mass with respect to the link frame. The `<inertia>` tag, on the other hand, needs to enclose the following 6 tags:
+
+ * `<ixx>` (Moment of Inertia)
+
+ * `<iyy>` (Moment of Inertia)
+
+ * `<izz>` (Moment of Inertia)
+
+ * `<ixy>` (Product of Inertia)
+ 
+ * `<ixz>` (Product of Inertia)
+ 
+ * `<izy>` (Product of Inertia)
+
+This proposal suggests the addition of an `auto` parameter for the `<inertia>` tag that would tell `libsdformat` to calculate Inertia matrix values automatically for the respective link. 
+
+Usage example:    
+```
+<inertia auto=”true” />
+```
+
+Addition of a `//link/collision/material_density` tag is also suggested. This density value would be used to calculate the inertial parameters of the respective collision geometry. Adding this as part of the `<collision>` tag would allow a user to simulate links with different material types for different collisions. By default, the value of density would be set equal to that of water which is 1 kg/m^3.  
+
+## Proposed Implementation
+Below are some key architectural considerations for the implementation of this feature:
+
+ *  The parsing of the proposed SDFormat elements and the Moment of Inertia calculations for primitive geometries(Box, Cylinder, Sphere, Ellipsoid and Capsule) can be developed as an integral part of libsdformat. This would help enable all simulators that rely on SDFormat to utilize this feature and not limit it to just Gazebo.
+
+ * In case of 3D meshes being used as geometries, a modular architecture can be followed where the user is free to develop and use their own Moments of Inertia Calculators. The default approach for handling MOI calculations of 3D meshes for Gazebo is proposed below and would rely on [Voxelization of Meshes](#inertia-matrix-calculation-with-voxelization-for-3d-mesh).
```

Comment:
```suggestion
 * In case of 3D meshes being used as geometries, a modular architecture can be followed where the user can integrate a custom Moments of Inertia calculator. Though for a default implementation of MOI calculations of 3D meshes [Voxelization of Meshes](#inertia-matrix-calculation-with-voxelization-for-3d-mesh) is proposed 
```
see section [cite].

Reply:
Fixed in 0a09b7da1822bb893b2b4aafce5d3a8c27874ff2